apps/api/src/domain/models/digital-text/entities/digital-text-page.entity.ts

```diff
@@ -10,14 +18,52 @@ export default class DigitalTextPage extends BaseDomainModel {
     })
     readonly identifier: PageIdentifier;
 
+    @NestedDataType(MultilingualText, {
+        label: 'content',
+        description: '(multilingual text) content for this page',
+        isOptional: true,
+    })
+    /**
+     * This is optional because users sometimes register the existence of a page
+     * by identifier (page number), even though they do not have access to or
+     * do not want to invest the time in adding the content at that point in
+     * time.
+     */
+    readonly content?: MultilingualText;
+
     constructor(dto: DTO<DigitalTextPage>) {
         super();
 
         if (!dto) return;
 
-        const { identifier } = dto;
+        const { identifier, content } = dto;
 
         // Note that this is just a string (stored by value not reference), so there is no need to clone or build an instance
         this.identifier = identifier;
+
+        this.content = !isNullOrUndefined(content) ? new MultilingualText(content) : null;
+    }
+
+    addContent(text: string, languageCode: LanguageCode): ResultOrError<DigitalTextPage> {
+        if (this.hasContent()) {
+            return new CannotOverwritePageContentError(this.identifier, this.content);
+        }
+
+        /**
+         * TODO Validate invariants as in `safeClone`.
+         */
+        return this.clone<DigitalTextPage>({
```

Comment:
This reminds me that I've been thinking for a while about having an `@UpdateMethod` decorator that wraps in the safeClone behaviour or even makes this behaviour configurable.